main.go

```diff
@@ -120,3 +120,34 @@ func getEnvVars(vars map[string]string) (map[string]string, error) {
 	}
 	return result, nil
 }
+
+func prCheckEarlyExit(envVars map[string]string) {
+	// indicates PR check when set
+	prevBundleSha := os.Getenv("PREVIOUS_BUNDLE_SHA")
```

Comment:
from this line i assume your strategy for the runners.sh in a-i is that you pass this env var if the MR supports it, so if only data or resource files change?

Reply:
Correct. Which I believe is okay for this functionality? 
The only attributes being queried are `codeComponent.gitlabSync` and `saasFile.resourceTemplate`. Both of which to my understanding solely exist within `data/`.

Reply:
thx